### ISSUE TITLE
base value doesn't show in Force_plot 

### ISSUE THREAD

**Issue by Twinkle123321** [shap/shap#934]
Hi,
I use LightGBM in python API 
base value doesn't show
however,
in  LightGBM's original API
it shows 

code is as follows：
        shap.force_plot(explainer.expected_value,shap_values[i,:],X.iloc[i,:],matplotlib=True,figsize=(23,3))

I'm appreciated if you could give me some suggestions.
![1](https://user-images.githubusercontent.com/32992310/70418847-2c164280-1a9f-11ea-8d41-ff9f87ed5fce.jpg)

**Comment by slundberg** [comment]
I am not sure I fully get what went wrong. Could you share the full code snippet that works and that doesn't work?

**Comment by Twinkle123321** [comment]
thanks for your reply
background:
X_train.shape is 464368 and label is unbalanced {0:1,1:13} so I do some sampling, and the param of shap.TreeExplainer in "data" is 100 to 1000 so test_size=0.0005
code is as follows:
clf2=lgb.LGBMClassifier(learning_rate=0.2,class_weight={0: 1, 1: 3}) 
 X_train_sam, X_test_sam, y_train_sam, y_truth_sam = train_test_split(X_train,y_train,test_size=0.0005, shuffle=True, random_state=2019,stratify=y_train)
explainer = shap.TreeExplainer(clf2, data=X_test_sam, model_output='probability')
shap_values = explainer.shap_values(X_test)
shap.force_plot(explainer.expected_value,shap_values[i,:],X_test.iloc[i,:],matplotlib=True,figsize=(23,3))

**Comment by slundberg** [comment]
So in that case you don't see the base value show up on the plot? What is the output of `explainer.expected_value`?

Also you might try the new `shap.waterfall_plot`

**Comment by Twinkle123321** [comment]
the output of  explainer.expected_value(base value) is 0.14
it can be outputted by printing,so I add it manully in  single shap figure.

I have another question in shap_values 
when explainer = shap.TreeExplainer(clf2)
 explainer.expected_value is 2-d ,that is [3.2062222020769826, -3.2062222020769826]
I guess 3.206 represents class 0,-3.206 represents class 1,is that true?
or which one should be chosen as base value?
there is a same problem in shap_values.
using
shap.summary_plot(shap_values[0])
shap.summary_plot(shap_values[1])
can get the opposite conclusion.
I'm still looking forward to your rely,thanks!
